examples/ios_app/test/fixtures/bwb_spec.json

```diff
@@ -1459,10 +1633,20 @@
                     "Utils/Foo.m"
                 ]
             },
-            "is_swift": false,
+            "is_swift": true,
```

Comment:
Utils is not a swift target

Reply:
I wanted to talk about this. What does `is_swift` mean for you? Before this PR, it was a target that provided `SwiftInfo`. I updated the logic to only mark it as true if it also directly provided a module. What do you think it should be?

Reply:
It means that it is compiling Swift code. In Xcode there are certain things that only make sense for a "Swift target", though we aren't properly doing some of that stuff today (like conditionally setting the last swift migration attributes).

Basically `swift_module.swift` needs to not be `None`.

Reply:
OK. So, I will update `targets.get_swift_info` to do the following:

1. Check for the existence of `SwiftInfo`.
2. Check if any of the `direct_modules` has its `swift` attribute set.

Does that work for you?

Reply:
Yep!